cwt.go

```diff
@@ -0,0 +1,20 @@
+package cose
+
+// https://www.iana.org/assignments/cwt/cwt.xhtml
+const (
+	CWTClaimIssuer         int64 = 1
+	CWTClaimSubject        int64 = 2
+	CWTClaimAudience       int64 = 3
+	CWTClaimExpirationTime int64 = 4
+	CWTClaimNotBefore      int64 = 5
+	CWTClaimIssuedAt       int64 = 6
+	CWTClaimCWTID          int64 = 7
+	CWTClaimConfirmation   int64 = 8
+	CWTClaimScope          int64 = 9
+
+	// TODO: the rest upon request
```

Comment:
I don't think it is smart to add the whole registry in the first PR.

Reply:
Based on the above, it looks like we have the minimal code needed to proceed.

Reply:
We should add what is bare minimum required, and then extend it when a new point needs to be registered!

Reply:
> [...] and then extend it when a new point needs to be registered!

Sorry, I have probably missed the right conversations, but I am not sure I get what the strategy is.  There are 10's of claims already registered.  What is the criterion for promoting them from the [CWT Claims](https://www.iana.org/assignments/cwt/cwt.xhtml#claims-registry) registry to `const`s in the `cose` package?